commands/start.js

```diff
@@ -26,28 +24,24 @@ function startCmd(opts) {
 	}
 }
 function printRoutes(srcPath, port) {
-	globby('**/', {cwd: srcPath})
+	/**
+	 * pattern to match valid file names
+	 * @example HTTP method: post.json
+	 * @example middleware: post.js
+	 */
```

Comment:
prefer to remove this jdoc-style comment here, it's pretty straightforwards what the next line is doing anyways